srcpkgs/mininet/template

```diff
@@ -1,16 +1,19 @@
 # Template file for 'mininet'
 pkgname=mininet
-version=2.2.2
-revision=2
-build_style=python2-module
-hostmakedepends="help2man python-setuptools"
-depends="bash inetutils-telnet iperf iputils libcgroup-utils net-tools openvswitch socat"
+version=2.3.0
+revision=1
+build_style=python3-module
+hostmakedepends="help2man python3-setuptools"
+depends="bash ethtool inetutils-telnet iperf iproute2 iputils libcgroup-utils
+ net-tools openssh openvswitch psmisc python3-tkinter socat xbitmaps xhost xterm"
 short_desc="Emulator for rapid prototyping of Software Defined Networks"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="classabbyamp <dev@kb6.ee>"
 license="custom:MIT-like"
 homepage="http://mininet.org/"
 distfiles="https://github.com/mininet/${pkgname}/archive/${version}.tar.gz"
-checksum=d0aed2ea7a9096ae975694a4b3d0995259ef79268dd8888ba8be28601c100db5
+checksum=1b16ee53ddb9a34a3751a77517643bda2286fc31be123209e84884d0697107b0
+# tests require root
+make_check=no
 
 post_extract() {
 	sed -i 's/^\tcc /\t$(CC) /' Makefile
```

Comment:
let's use vsed here

Reply:
fixed :)

Reply:
And it revealed that we don't even need this:
```
=> WARNING: mininet-2.3.0_1: vsed: regex "s/^	cc /	$(CC) /" didn't change file "Makefile"
```